cmd/arc_summary/arc_summary.py

```diff
@@ -705,8 +705,8 @@ def _l2arc_summary(Kstat):
         )
         sys.stdout.write("\n")
 
-        if arc["l2_arc_evicts"]['lock_retries'] + \
-                arc["l2_arc_evicts"]["reading"] > 0:
+        if arc["l2_arc_evicts"]['lock_retries'] != '0' or \
+           arc["l2_arc_evicts"]["reading"] != '0':
```

Comment:
My only minor concern with flipping the logic as you suggest is that this script originally came from [FreeNAS](https://github.com/freenas/freenas/blob/master/gui/tools/arc_summary.py) and making this change would complicate syncing up with future changes.  That said, this tool hasn't seen significant development on FreeNAS in the last few years and we've already needed to modify it for Linux. 

@jstenback let's go ahead with the minimal fix as you've proposed it in this PR.  If someone wants to further extend this tool we can look in to restructuring it then.  Thanks for the offer of restructuring this, but let's hold off for now.